utils/utils.js

```diff
@@ -153,6 +158,34 @@ function stringifyFields(obj, fieldsToStringify) {
   );
 }
 
+function isNestingAddress(address) {
+  return (
+    address.indexOf(NESTING_ADDRESS_PREFIX) === 0 && address.length === NESTING_ADDRESS_LENGTH
+  );
+}
+
+function getCollectionIdFromNestingAddress(address) {
+  if (!isNestingAddress(address)) return null;
+
+  const collectionString = address.slice(
+    NESTING_ADDRESS_PREFIX.length,
+    NESTING_ADDRESS_PREFIX.length + 8,
+  );
+
+  return parseInt(collectionString, 16) || null;
+}
+
+function getTokenIdFromNestingAddress(address) {
+  if (!isNestingAddress(address)) return null;
+
+  const tokenString = address.slice(
+    NESTING_ADDRESS_PREFIX.length + 8,
+    NESTING_ADDRESS_PREFIX.length + 25,
+  );
+
+  return parseInt(tokenString, 16) || null;
+}
```

Comment:
It looks like we always should use these two functions together? Maybe it makes sence to have one function for parsing nesting address, for example `parseNestingAddress(fullAddress: string): { collectionId: string, tokenId: string }`?